bundles/org.openhab.binding.smsmodem/README.md

```diff
@@ -0,0 +1,129 @@
+
+## Discovery
+
+There is no discovery process for *smsmodembridge* thing.
+A *smsconversation* thing will be discovered and added to the inbox everytime the modem should receive a SMS by a new sender.
+
+## Thing Configuration
+
+The *smsmodembridge* thing requires at least two parameters to work properly (serialPortOrIP, baudOrNetworkPort).
+Depending on the nature of the connection (direct serial modem, or serial over network), this two fields will be used differently :
+
+| field | direct serial modem   | serial over network                  |
+|-------|--------------------------|-----------------------------------|
+|serialPortOrIP| The serial port to access (eg. /dev/tty/USBx) | IP address of the computer hosting the ser2net service|
+|baudOrNetworkPort| Baud rate                                 | The network port of the ser2net service |
+
+The other parameters are optional :
+
+| field | description     |
+|-------|--------------------------------------|
+|simPin          |If your sim card is protected, fill this field with the PIN code|
+|pollingInterval| Delay between two checks for new message|
+|delayBetweenSend|Delay to wait between two messages post (could be necessary for slow modem)|
+
+```
+Bridge smsmodem:smsmodembridge:adonglename [ serialPortOrIP="/dev/ttyUSB0", baudOrNetworkPort="19200", enableAutoDiscovery="true" ]
+```
+
+The *smsconversation* thing is just a shortcut to address/receive messages with a specific msisdn. It is not mandatory to use the binding, as you can use action and trigger channel to send/receive a message once the smsmodem bridge is configured.
+
+| field | description               |
+|-------|--------------------------|
+| recipient | The msisdn of the phone you want to discuss with.|
+| deliveryReport | If enabled, ask the network for a delivery report (default false)|
+| encoding | The encoding to use when sending the message (either Enc7, Enc8, EncUcs2, EncCustom, default is Enc7). EncUcs2 is good for non latin character, but SMS character size limit is then reduced|
+
+
+```
+Thing smsmodem:smsconversation:aconversationname [ recipient="XXXXXXXXXXX", deliveryReport="true" ]
+```
+
+## Channels
+
+The *smsconversation* supports the following channels :
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| receive | String| The last message received |
+| send | String| A message to send |
+|deliverystatus| String| Delivery status (either UNKNOWN, QUEUED, SENT, PENDING, DELIVERED, EXPIRED, or FAILED). Several status are only possible if the delivery report parameter is enabled|
+
+## Trigger channels
+
+The *smsmodembridge* has the following trigger channel :
+| Channel ID          | event                      |  
+|---------------------|----------------------------|
+|receivetrigger| The msisdn and message received (concatened with the '\|' character as a separator)|
```

Comment:
"trigger" is redundant and can therefore be removed.
```suggestion
|receive| The msisdn and message received (concatened with the '\|' character as a separator)|
```

Reply:
There is already a "receive" channel. So you are saying, there won't be a naming issue ?

Reply:
Ah, seems like I overlooked this. Then, everything is fine.